src/2-run-ipfs-on-a-vps/index.md

```diff
@@ -0,0 +1,79 @@
+---
+baseurl: ..
+template: tmpl/layouts/post.html
+breadcrumbs:
+  - {name: "2-run-ipfs-on-a-vps", link: "./" }
+id: 2-run-ipfs-on-a-vps
+date: 2015-11-02
+title: Run IPFS latest on a VPS
+author: Kyle Drake
+collection: posts
+---
+
+The best way to provide content using [IPFS](https://ipfs.io) is to run your own IPFS node. You can do this by running an IPFS node on your personal computer, but that will only work as long as your computer is running. For hosting content with a higher availability, you need to run an IPFS node in a datacenter. That ensures your content is always online and available to other nodes on the network.
```

Comment:
since so much of the pitch for ipfs is to not necessarily require the datacenter, would be good to say this is _useful_ but not _needed_. basically, something more like:

> The best way to provide content using [IPFS](https://ipfs.io) is to run your own IPFS node. You can do this by running an IPFS node on your personal computer, but that will only work as long as your computer is running. For users running mostly from laptops or with bandwidth constraints, it is useful to run IPFS nodes in a datacenter, and _pinning_ the content there too. This ensures your content is replicated, online, and available to other nodes on the network.